doc/releases/changelog-0.23.0.md

```diff
@@ -140,59 +195,118 @@
-    [(#2362)](https://github.com/PennyLaneAI/pennylane/pull/2362)
   - New functions are added for building fermionic and qubit observables
-    [(#2230)](https://github.com/PennyLaneAI/pennylane/pull/2230)
   - A new module is created for hosting openfermion to pennylane observable conversion functions
-    [(#2199)](https://github.com/PennyLaneAI/pennylane/pull/2199)
-    [(#2371)](https://github.com/PennyLaneAI/pennylane/pull/2371)
   - Expressive names are used for the Hartree-Fock solver functions
-    [(#2272)](https://github.com/PennyLaneAI/pennylane/pull/2272)
   - These new additions are added to a feature branch
-    [(#2164)](https://github.com/PennyLaneAI/pennylane/pull/2164)
   - The efficiency of computing molecular integrals and Hamiltonian is improved
-    [(#2316)](https://github.com/PennyLaneAI/pennylane/pull/2316)
   - The qchem and new hf modules are merged
-    [(#2385)](https://github.com/PennyLaneAI/pennylane/pull/2385)
   - The 6-31G basis set is added to the qchem basis set repo
-    [(#2372)](https://github.com/PennyLaneAI/pennylane/pull/2372)
   - The dependency on openbabel is removed
-    [(#2415)](https://github.com/PennyLaneAI/pennylane/pull/2415)
   - The tapering functions are added to qchem
-    [(#2426)](https://github.com/PennyLaneAI/pennylane/pull/2426)
   - Differentiable and non-differentiable backends can be selected for building a Hamiltonian
-    [(#2441)](https://github.com/PennyLaneAI/pennylane/pull/2441)
   - The quantum chemistry functionalities are unified
-    [(#2420)](https://github.com/PennyLaneAI/pennylane/pull/2420)
-    [(#2465)](https://github.com/PennyLaneAI/pennylane/pull/2465)
-    [(#2454)](https://github.com/PennyLaneAI/pennylane/pull/2454)
 
-* Adds a MERA template.
+<h4>Pattern matching optimization 🔎 💎 </h4>
+
+* Added an optimization transform that matches pieces of user-provided identity
+  templates in a circuit and replaces them with an equivalent component.
+  [(#2032)](https://github.com/PennyLaneAI/pennylane/pull/2032)
+
+  First let's consider the following circuit where we want to replace sequence of two ``pennylane.S`` gates with a
+  ``pennylane.PauliZ`` gate.
+
+  ```python
+  def circuit():
+      qml.S(wires=0)
+      qml.PauliZ(wires=0)
+      qml.S(wires=1)
+      qml.CZ(wires=[0, 1])
+      qml.S(wires=1)
+      qml.S(wires=2)
+      qml.CZ(wires=[1, 2])
+      qml.S(wires=2)
+      return qml.expval(qml.PauliX(wires=0))
+  ```
+
+  Therefore we use the following pattern that implements the identity:
+
+  ```python
+  with qml.tape.QuantumTape() as pattern:
+      qml.S(wires=0)
+      qml.S(wires=0)
+      qml.PauliZ(wires=0)
+  ```
+
+  For optimizing the circuit given the given following template of CNOTs we apply the `pattern_matching`
+  transform.
+
+  ```pycon
+  >>> dev = qml.device('default.qubit', wires=5)
+  >>> qnode = qml.QNode(circuit, dev)
+  >>> optimized_qfunc = qml.transforms.pattern_matching_optimization(pattern_tapes=[pattern])(circuit)
```

Comment:
Is this correct? Should it be `pattern_matching` or `pattern_matching_optimization`?

Reply:
Yep, this one is `pattern_matching_optimization`.